### PR TITLE
Improve blog page speed

### PR DESCRIPTION
**Pull request by stefan9541**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -457,7 +457,7 @@ app.prepare().then(() => {
     });
   });
 
-  server.get('/blog/:name', (req, res) => app.render(req, res, '/post', { name: req.params.name }));
+  // server.get('/blog/:name', (req, res) => app.render(req, res, '/blog', { name: req.params.name }));
```
Comment (epaminond) [inline server/index.js:460]: Why would you leave commented out code?

File: helpers/date-parser.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/* eslint-disable camelcase */
+const months = [
```
Comment (epaminond) [inline helpers/date-parser.js:2]: Why migrating from Moment.js?

Comment (stefan9541) [reply helpers/date-parser.js:2]: use a library weighing 50kb in order to render 1 paragraph, in my opinion, this is not reasonable, and react-moment used only on blog and post pages, so in other pages where we really need moment it still stays

File: server/index.js (hunk 1 above)
Comment (stefan9541) [reply server/index.js:460]: perhaps I forgot to delete it)

File: components/layout/head.js [hunk 3]
```diff
@@ -80,9 +80,9 @@ const LayoutHead = (props) => {
     description = currentPoint.metaDescription || currentPoint.description;
   }
 
-  if (currentURL.pathname === '/post') {
-    currentURL.pathname = router.asPath.replace('#', '?').split('?')[0];
-  }
+  // if (currentURL.pathname === '/post') {
+  //   currentURL.pathname = router.asPath.replace('#', '?').split('?')[0];
+  // }
```
Comment (epaminond) [inline components/layout/head.js:85]: Do you really need to keep this commented out code? It'll still stay in git.

File: pages/blog/[id].js [hunk 4]
```diff
@@ -453,6 +436,42 @@ export default class Post extends React.Component {
   }
 }
 
+export async function getStaticPaths() {
+  // Call an external API endpoint to get posts
```
Comment (epaminond) [inline pages/blog/[id].js:440]: Not sure you need comments like this.
The code should be self-documented. It'd be great for comments to explain "why?" not "what?" or "how?"

File: server/index.js (hunk 1 above)
Comment (epaminond) [reply server/index.js:460]: Perhaps. )
Let's delete it than.

File: helpers/date-parser.js (hunk 2 above)
Comment (epaminond) [reply helpers/date-parser.js:2]: OK, makes sense.

Regarding code itself, I'd suggest you to use nice snippets from https://github.com/you-dont-need/You-Dont-Need-Momentjs#string--date-format

**Pull request closed without merging**